src/App/components/ActionLink/index.js

```diff
@@ -7,31 +7,39 @@ const ActionLink = props => {
     return (
         <>
             {otherProps.newTab ?
-                <a className="action-link action-link-new-tab" href={link ? link : "#"} rel="noopener noreferrer" target="_blank">{"\n"}
+                <a className={`action-link${otherProps.multiline ? " multiline" : ""} action-link-new-tab${otherProps.disabled ? " disabled" : ""}`} href={link ? link : "#"} rel="noopener noreferrer" target="_blank">{"\n"}
                     <ActionLinkContent {...otherProps} />
                 </a>
                 :
-                <a className="action-link" href="#" >{"\n"}
+                <a className={`action-link${otherProps.multiline ? " multiline" : ""}${otherProps.disabled ? " disabled" : ""}`} href="#" >{"\n"}
                     <ActionLinkContent {...otherProps} />
                 </a>
             }
         </>
     );
 };
 
-const ActionLinkContent = ({ linkText, badge, smallText, multiline, newTab }) => (
+const ActionLinkContent = ({ linkText, badge, smallText, multiline }) => (
     <>
-        {newTab ? <><i className="material-icons" aria-hidden="true">open_in_new</i>{"\n"}</> : null}
-        {badge ? <><span className={`badge ${badge.type}`}>{badge.text}</span>{"\n"}</> : null}
         {multiline ?
             <span className="action-link-multiline">{"\n"}
-                {linkText}{"\n"}
-                {smallText ? <span className="small-text">{smallText}</span> : null }{"\n"}
+                <div>
+                    <div className="d-flex flex-row">
+                        <h4 className="mb-0 mr-1">{linkText}</h4>{"\n"}
```

Comment:
Looking at it now, h4 gives the wrong font-size. And I also don't think it would be semantic correct to use a heading tag here anyways. But this is my fault, forgive me 😢

Reply:
Haha, I forgive you 🙏